frontend/.husky/pre-commit

```diff
@@ -3,3 +3,9 @@
 
 cd frontend
 npx --no-install lint-staged
+
+cd ..
+echo "Running black..."
+black --check .
+echo "Running mypy..."
+mypy .
```

Comment:
Ideally you'd add these to [`.lintstagedrc`](https://github.com/mozilla/fx-private-relay/blob/main/frontend/.lintstagedrc.js) to use [lint-staged](https://www.npmjs.com/package/lint-staged), so they only run against staged changes. You'd add something like this:

```js
  "*.py": "black",
  "*.py": "mypy",
```